config/locales/en.yml

```diff
@@ -3664,8 +3665,8 @@ en:
           routing_number: Routing Number
           withdraw_amount: How much do you authorize to be withdrawn from your account? (Your total amount due is&#58; $%{owed_amount}.)
         edit:
-          direct_deposit: Direct deposit (fastest)
-          mail: Mail my payment (slower)
+          direct_deposit: By direct deposit (fastest)
+          mail: By mail (slower)
```

Comment:
recognizing this is a cross-state change, asking if this is what we want for all states:

[Q Here](https://codeforamerica.atlassian.net/browse/FYST-1529?focusedCommentId=14360)